airbyte-integrations/connectors/source-snowflake/src/main/java/io.airbyte.integrations.source.snowflake/SnowflakeSourceOperations.java

```diff
@@ -25,6 +41,23 @@ protected void putDouble(final ObjectNode node, final String columnName, final R
     }
   }
 
+  @Override
+  public JDBCType getFieldType(final JsonNode field) {
+    try {
+      final String typeName = field.get(INTERNAL_COLUMN_TYPE_NAME).asText().toLowerCase();
+      return "TIMESTAMPLTZ".equalsIgnoreCase(typeName)
```

Comment:
is there a test case for a `timestampltz` column?

Reply:
yup, please take a look [here](https://github.com/airbytehq/airbyte/blob/04236ac40442e0a287a24f3a17a68f0aed8f5bd2/airbyte-integrations/connectors/source-snowflake/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/SnowflakeSourceDatatypeTest.java#L283)

Reply:
nice, I somehow didn't find that when I did a cmd-f yesterday